scripts/openroad/or_droute.tcl

```diff
@@ -22,4 +22,9 @@ if {[catch {read_def $::env(CURRENT_DEF)} errmsg]} {
     exit 1
 }
 
-tr::detailed_route_cmd $::env(tritonRoute_tmp_file_tag).param
+set_thread_count $::env(ROUTING_CORES)
+
+detailed_route -param $::env(tritonRoute_tmp_file_tag).param
```

Comment:
All the options in -param are now available as command options so there is no need to use the param file anymore.